recipes/TIES20/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "TIES20" %}
+{% set version = "20.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/UCL-CCS/TIES/archive/refs/tags/{{ version }}.tar.gz
+  sha256: a8b74cd61e66d4fbb8b4565cc18eaa6ac2280743b8764c57bacd67ec4158882a
+
+build:
+  number: 0
+  skip: True  # [win or osx and py==310]
+  script: {{ PYTHON }} -m pip install . -vv
+  entry_points:
+    - ties = ties:cli.command_line_script
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
```

Comment:
Is the CXX compiler actually used? If not, remove it. Cython compiles to c code.

Reply:
Thank you, resolved.